Dockerfile

```diff
@@ -32,7 +33,7 @@ COPY cross/add_cross_architecture_ports.sh /
 RUN /add_cross_architecture_ports.sh
 
 # install the cross compiler
-RUN apt-get update && apt-get install -y cmake crossbuild-essential-${BUILD_ARCHITECTURE} file
+RUN apt-get update && apt-get install -y crossbuild-essential-${BUILD_ARCHITECTURE} file
```

Comment:
If we are just removing the `cmake` here, which component or where in the pipeline do we install it? 
Is there any README.md need to be updated because of this change?

Reply:
These are all installed by https://github.com/usdot-fhwa-stol/carma-builds/blob/develop/scripts/install_build_tools.sh. Do you think it would be helpful to mention this explicitly in the README or is the script naming sufficient

Reply:
Ah I think that is okay. It maybe beginner question, but looking at the `install_build_tools.sh` script, I wonder how does installing x86-64 specific cmake work in other architectures?
```
wget https://github.com/Kitware/CMake/releases/download/v${CMAKE_VERSION}/cmake-${CMAKE_VERSION}-linux-x86_64.sh
wget https://github.com/Kitware/CMake/releases/download/v${CMAKE_VERSION}/cmake-${CMAKE_VERSION}-linux-x86_64.tar.gz
chmod u+x cmake-${CMAKE_VERSION}-linux-x86_64.sh
./cmake-${CMAKE_VERSION}-linux-x86_64.sh --skip-license --exclude-subdir --prefix=/usr/local
rm cmake-${CMAKE_VERSION}-linux-x86_64.sh
rm cmake-${CMAKE_VERSION}-linux-x86_64.tar.gz
``` 
Do the scripts from `cross` go and fix/adjust after the x86-64 installation is done to make it cross compilable? 
Has it been tested to work for the original issue you were encountering?

Reply:
Talked offline about it, looks good to me